content/departments/engineering/product/process/index.md

```diff
@@ -38,6 +38,30 @@ See [strategy](../../../../strategy-goals/index.md).
 - [Pricing and tiering/packaging](gtm/pricing.md)
 - [Licensing](gtm/licensing.md)
 
+#### Experimenting on sourcegraph.com
+
+We welcome experimentations on sourcegraph.com. It should be very easy to experiment on .com, send a tweet or email campaign out to get some traffics, and get early feedback in.
+
+The bar for an experiment on .com is the following:
+
+- Break nothing (new UIs and public data APIs are low risk, anything involving sensitive data or critical APIs is high risk)
+- Don't disrupt the main flows
+- Add basic usage metrics to know what's going on
+
+In particular an experiment does not require:
+
+- Discoverability
+- A polished design
+- Approvals, as long as it meets the criteria above
+- A/B testing, although A/B testing can be used
+
+When you experiment on .com you commit to:
+
+- Following our [feature labelling](https://docs.sourcegraph.com/admin/beta_and_experimental_features) guidelines
+- Maintaining and supporting it (until removed or removed)
+- Working on progressing the feature to Beta/GA or removing it no more than 1 month after the experiment started.
```

Comment:
+1 to having a time limit. I do wonder if we should clarify that progressing to Beta/GA doesn't mean that it has to be Beta/GA within a month. Rather, it means that the experiment owner must, at the least, be actively iterating on the experiment.

Reply:
Good point. That's what I meant, but definitely unclear in this sentence.